Makefile

```diff
@@ -4,6 +4,19 @@ build:
 	@go generate ./...
 .PHONY: build
 
+# Install the build locally
+install:
+	@go install ./...
+	@echo "==> Installed up ${GOPATH}/bin/up"
+.PHONY: install
```

Comment:
Otherwise LGTM! But we should omit bin/bin_assets.go and bindata from this commit, I think it's just the mtime changing